tests/conftest.py

```diff
@@ -4,13 +4,6 @@
 from __future__ import unicode_literals
 
 import pytest
-from py._path.local import LocalPath
-
-
-@pytest.fixture(autouse=True)
-def cwd():
```

Comment:
I was getting errors about the `py` module (`ModuleNotFoundError: No module named 'py._path'; 'py' is not a package` showed up in itest_focal and itest_jammy, but not for xenial and bionic), but it doesn't seem like this fixture is actually doing anything, since tests and itests still pass?

Reply:
this is still important to ensure that the tests don't depend on pwd -- you can use `os.chdir` to do this instead

Reply:
Added it back